ipapython/ipautil.py

```diff
@@ -142,6 +153,7 @@ def __init__(self, addr, match_local=False, parse_netmask=True,
 
         if isinstance(addr, CheckedIPAddress):
             self.prefixlen = addr.prefixlen
+            self._net = addr._net
```

Comment:
`self._net` is already set by the `super(CheckedIPAddress, self).__init__(addr)` call above.